nano/core_test/work_pool.cpp

```diff
@@ -178,10 +178,10 @@ TEST (work, eco_pow)
 	std::future<std::chrono::nanoseconds> future2 = promise2.get_future ();
 
 	std::thread thread1 (work_func, std::ref (promise1), std::chrono::nanoseconds (0));
-	std::thread thread2 (work_func, std::ref (promise2), std::chrono::nanoseconds (100000));
+	std::thread thread2 (work_func, std::ref (promise2), std::chrono::nanoseconds (10000000));
```

Comment:
Maybe `std::chrono::milliseconds (10)` is a bit more readable. Other than that, LGTM and the test now survives 1000 iterations.

Reply:
Good point, done